README.md

```diff
@@ -77,7 +73,7 @@ Will have future changes in the next days!
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-<p> Checkout my capstone <a href="https://enmanuelbayen.github.io/Capstone-Web-page/" target="_blank" rel="noopener noreferrer"> here!</a> </p>  
+<p> Checkout my capstone <a href="#" target="_blank" rel="noopener noreferrer"> here!</a> </p>  
```

Comment:
- [ ] Kindly remove this section entirely or add a working deployment link for this project to make your readme file more professional.